pkg/controller/provider/container/ovirt/client.go

```diff
@@ -33,14 +33,34 @@ type Client struct {
 	// Raw client.
 	client *libweb.Client
 	// Secret.
-	secret *core.Secret
+	secret                *core.Secret
+	created               time.Time
+	clientTimeout         time.Duration
```

Comment:
@ahadas the question in the original PR was about clientTimeout and accessTokenExpiration?

Reply:
no, about 'created' and 'clientTimeout'

Reply:
ah ok, so I'm not sure how to unify them, I need `created` to know when the "timer" was started

Reply:
it could well be that I'm missing something - both `created` and `clientTimeout` are set in collector::New - can't we set an expiration field that is set there to Now() + clientTimeout instead?

Reply:
ah, I forgot about it, I only set it there because I thought the zero value of time.Time is not useful but it doesn't matter actually, the actual creation time is set in line 135

I changed `created` to `clientExpiration` and made it a calculated field so now the check looks nicer